boards/ti/lp_em_cc2340r5/lp_em_cc2340r5.dts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024 Texas Instruments Incorporated
+ * Copyright (c) 2024 BayLibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <ti/cc2340r5.dtsi>
+
+/ {
+	model = "LP_EM_CC2340R5";
+	compatible = "ti,lp_em_cc2340r5";
+
+	chosen {
+		zephyr,sram = &sram0;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <48000000>;
+};
```

Comment:
is this a board-level setting?

Reply:
That would make more sense at SoC-level indeed.